components/grid/export/events.md

```diff
@@ -141,13 +141,13 @@ To export a hidden (the Visible attribute set to `false`) column you can manuall
 
 * `Data` - `IEnumerable<object>` - assign a custom collection of data to be exported to CSV, for example only the selected items in the Grid. 
 
-* `Columns` - `List<ExcelExportableColumns` - a collection of all exportable columns in the Grid (the columns that have a defined `Field` and are visible). You can customize the following attributes of the Grid column before exporting it into Excel:
+* `Columns` - `List<ExcelExportableColumns>` - a collection of all exportable columns in the Grid (the columns that have a defined `Field` and are visible). You can customize the following attributes of the Grid column before exporting it into Excel:
 
-    * `Width` - define the width of the column **in pixels**.
-    * `Title` - define the column title to be shown in the Excel file header. 
-    * `NumberFormat` - provide an Excel-compatible number/date format
+    * `Title` - define the column title to be shown in the Excel file header.
     * `Field` - set the data bound field of the column.
     
+>important `Width` and `NumberFormat` properties are not applicable for CSV export and will be removed in **UI for Blazor 4.0**. For CSV formatting see [Format numbers and dates in the exported CSV file from the Grid]({%slug grid-kb-number-formatting-of-the-csv-export%}).
```

Comment:
Should be either "THE `Width` and `NumberFormat` properties are..." or "`Width` and `NumberFormat` are..."

```suggestion
>important `Width` and `NumberFormat` properties are not applicable for CSV export and will be removed in **UI for Blazor 4.0**. For CSV formatting see [Format numbers and dates in the exported CSV file from the Grid]({%slug grid-kb-number-formatting-of-the-csv-export%}).
```